examples/baselines/bc/bc.py

```diff
@@ -0,0 +1,408 @@
+            # is the terminal observation which has no actions
+            self.observations.append(trajectory["obs"][:-1])
+            self.actions.append(trajectory["actions"])
+            self.dones.append(trajectory["success"].reshape(-1, 1))
+            # print(trajectory.keys())
+        self.observations = np.vstack(self.observations)
+        self.actions = np.vstack(self.actions)
+        self.dones = np.vstack(self.dones)
+        assert self.observations.shape[0] == self.actions.shape[0]
+        assert self.dones.shape[0] == self.actions.shape[0]
+
+        if normalize_states:
+            mean, std = self.get_state_stats()
+            self.observations = (self.observations - mean) / std
+
+        # self.rewards = np.vstack(self.rewards)
+
+    def get_state_stats(self):
+        return np.mean(self.observations), np.std(self.observations)
+
+    def __len__(self):
+        return len(self.observations)
+
+    def __getitem__(self, idx):
+        action = torch.from_numpy(self.actions[idx]).float().to(device=self.device)
+        obs = torch.from_numpy(self.observations[idx]).float().to(device=self.device)
+        done = torch.from_numpy(self.dones[idx]).to(device=self.device)
+        return obs, action, done
+
+
+class Actor(nn.Module):
+    def __init__(self, state_dim: int, action_dim: int):
+        super(Actor, self).__init__()
+
+        self.net = nn.Sequential(
+            nn.Linear(state_dim, 256),
+            nn.ReLU(),
+            nn.Linear(256, 256),
+            nn.ReLU(),
+            nn.Linear(256, action_dim),
+        )
+
+    def forward(self, state: torch.Tensor) -> torch.Tensor:
+        return self.net(state)
+
+    @torch.no_grad()
+    def act(self, state, gpu = False):
+        if gpu:
+            return self(state)
+        state = torch.from_numpy(state).to(device="cuda:0")
+        return self(state).cpu().data.numpy()
+
+
+def save_ckpt(tag):
+    os.makedirs(f'{log_path}/checkpoints', exist_ok=True)
+    torch.save({
+        'actor': actor.state_dict(),
+    }, f'{log_path}/checkpoints/{tag}.pt')
+
+if __name__ == "__main__":
+    args = tyro.cli(Config)
+    set_seed(args.seed)
+
+    now = datetime.datetime.now().strftime("%y%m%d-%H%M%S")
+    tag = '{:s}_{:d}'.format(now, args.seed)
+    log_name = os.path.join(args.env, "BC", tag)
+    log_path = os.path.join(args.output_dir, log_name)
+
+
+    device = torch.device(args.device if torch.cuda.is_available() else "cpu")
+    control_mode = os.path.split(args.demo_path)[1].split(".")[2]
+
+
+    if args.wandb:
+        wandb_init(asdict(args))
+    ds = ManiSkillDataset(
+        args.demo_path,
+        device=device,
+        load_count=args.load_count,
+        normalize_states=args.normalize_states,
+    )
+    stats = ds.get_state_stats()
+
+    if args.video:
+       os.makedirs(os.path.join(log_path, args.video_path))
+    gpu = args.sim_backend == "gpu"
+    envs = make_eval_envs(args.env, args.num_envs, stats, control_mode,gpu)
```

Comment:
we should have a CLI argument to modify the max episode steps number to be higher from the command line. The code assumes 100 at the moment? You can replace max_traj_len with max_episode_steps as an argument. This should then also be passed to the environment creation tool.